lib/mapped-device.js

```diff
@@ -113,6 +113,9 @@ module.exports.MappedDevice = class MappedDevice {
     // assumed to belong together)
     let groups = this.groupCapabilities();
 
+    // some accessories have multiple services, so we sort the main service first
+    this.#maps.sort((x, y) => x.name === x.mainService ? -1 : y.name === y.mainService ? 1 : 0);
```

Comment:
@robertklep, given the way I've implemented, another logic would be to say that in case of multiple services, only one has `required.onoff` (see https://github.com/robertklep/name.klep.homekitty/pull/111/files#diff-a2618c39e5c62e09ab2ef031ebd30c76d1a7781f4f9dabce12c2276d39a2725eR5), and that this is the one which should come first. What do you think?